client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/features.js

```diff
@@ -0,0 +1,2 @@
+import { domMax } from 'framer-motion';
+export default domMax;
```

Comment:
We already have [framer-motion](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/animation/index.js) in `@wordpress/components`. Shall we use imported from there? It is being used in Gutenberg as [__unstableMotion](https://github.com/WordPress/gutenberg/search?q=__unstableMotion).

Reply:
Gutenberg only exports `AnimatePresence` and `motion` and it doesn't match our use case. So, I think it's okay to import components from `framer-motion` directly now.

Reply:
@arthur791004 In the latest commit I import the framer components lazily but the entry-stepper still increased in size [http://iscalypsofastyet.com/p/branch?branch=add/pa-list-animation,](http://iscalypsofastyet.com/p/branch?branch=add/pa-list-animation,) I think it is because the `m` import can not be used with React.lazy since it is not a component. What do you think?

Reply:
I think we could create the `AnimateList` component to wrap the all components from `framer-motion` and just lazy load this component.

For example,

```js
// animate-list.tsx
interface Props = {
  featureName: 'domMax' | 'domAnimation';
  children: ( m ) => JSX.Element;
}

const AnimateList = ({ featureName, children }) => (
  <LazyMotion features={ () => loadFeatures( featureName ) } strict>
    <AnimatePresence initial={ false }>
      { children( m ) }
    </AnimatePresence>
  </LazyMotion>
);

// usage
<AsyncLoad require="./animate-list">
  { ( m ) => sections.map( ( section ) => <m.li {...}>...</m.li> ) }
</AsyncLoad>
```